build.gradle

```diff
@@ -714,8 +715,7 @@ uiTestTargets.each { version ->
     if (project.hasProperty('headless')) {
       environment 'IDEPROBE_DISPLAY', 'xvfb'
       // TODO (VirtusLab/ide-probe#195): remove the need to set these env variables
-      environment 'IDEPROBE_TEST_SUITE', 'uiTest' + version
-      environment 'IDEPROBE_TEST_CASE', 'artifacts'
+      environment 'MY_CI_SCREENSHOTS_DIR', '/artifacts/uiTest' + version + '/screenshots'
```

Comment:
Actually... inside your local machine (MacOS/VM), are you able to store/retrieve these screenshots? My suspicion is that you won't be able to create `/artifacts` directory in the first place (and in CI it's only possible coz we run as root)

Reply:
are you able to run uiTest task without sudo ? I'm still getting IOException (I gave all permissions to folder in which execute this task - probably need to change permissions in other places too)

Reply:
I wouldn't rely on developers creating and `chmod`ing their `/artifacts` directory locally... just pick a directory that doesn't require root to create!

Reply:
Much better!